Library/Formula/aws-apigateway-importer.rb

```diff
@@ -0,0 +1,20 @@
+class AwsApigatewayImporter < Formula
+  desc "AWS API Gateway Importer"
+  homepage "https://github.com/awslabs/aws-apigateway-importer"
+  url "https://github.com/awslabs/aws-apigateway-importer/archive/aws-apigateway-importer-1.0.1.tar.gz"
+  sha256 "1aecfd348135c2e364ce5e105d91d5750472ac4cb8848679d774a2ac00024d26"
+
+  depends_on :java => "1.7+"
+  depends_on "maven" => :build
+
+  def install
+    ENV["_JAVA_OPTIONS"] = "-Duser.home=#{buildpath}"
+    system "mvn", "assembly:assembly"
+    libexec.install "target/aws-apigateway-importer-1.0.1-jar-with-dependencies.jar"
+    bin.write_jar_script libexec/"aws-apigateway-importer-1.0.1-jar-with-dependencies.jar", "aws-api-import"
+  end
+
+  test do
+    (`#{bin}/aws-api-import` =~ /Usage:\s+aws-api-import/) == 0
```

Comment:
Unfortunately, all the options either read from or modify stuff on an aws account.  `--help` is also the same as calling with no options, but it couldn't hurt to be more specific.